src/platform/intel/cavs/platform.c

```diff
@@ -145,7 +145,7 @@ EXT_MAN_PORT(
 ));
 #endif
 
-#if CONFIG_CANNONLAKE || CONFIG_ICELAKE || CONFIG_TIGERLAKE
+#if !CONFIG_SUECREEK && CAVS_VERSION >= CAVS_VERSION_1_8
```

Comment:
LPRO as the default clock not available for sue?

Reply:
(CONFIG_CANNONLAKE || CONFIG_ICELAKE || CONFIG_TIGERLAKE)
so probably no lpro